example/tests/test_models.py

```diff
@@ -274,3 +273,6 @@ def test_notification_should_be_filtered_with_related_object(self):
         self.assertEqual(Notification.objects.filter_with_related_object(self.author)[0], self.notification)
         # recipient is not related object
         self.assertFalse(Notification.objects.filter_with_related_object(self.recipient).exists())
+
+    def test_get_default_notification_model(self):
```

Comment:
This test is redundant now, as you are testing the whole logic in `test_helpers`. Instead, you should add test to `test_handlers.py` to verify, that handlers use this setting, i.e. that returned notification is the desired model.